features/built_in_matchers/include.feature

```diff
@@ -4,15 +4,15 @@ Feature: `include` matcher
 
     ```ruby
     expect("a string").to include("a")
-    expect("a string").to include("str")
+    expect("a string").to include(/a|str/).twice
     expect("a string").to include("str", "g")
     expect("a string").not_to include("foo")
 
     expect([1, 2]).to include(1)
     expect([1, 2]).to include(1, 2)
     expect([1, 2]).to include(a_kind_of(Integer))
     expect([1, 2]).to include(be_odd.and be < 10 )
-    expect([1, 2]).to include(be_odd)
```

Comment:
I was wondering why it was there at all; the example just before it seems more interesting. I can add it back if you want.

Reply:
Yes, please. It's a very basic example of using an argument matcher. This file is published as docs, so it would be easier to understand possible usages with this being kept.

Reply:
Done